packages/content/src/hub.ts

```diff
@@ -269,8 +297,13 @@ export function datasetToItem(dataset: DatasetResource): IItem {
     id: itemId,
     owner: owner as string,
     orgId,
-    created: (created || createdAt) as number,
-    modified: (modified || updatedAt) as number,
+    created: created as number,
```

Comment:
I sorta thought createdAt came from the hub api but i don't see it. Are we sure we will never have createdAt?

Reply:
Hmm thinking more about this, i guess created at was from our composer service.

Reply:
it comes from the model serializer (or composer). I've tested this locally sending models that come out of composer in the app's `datasetModelToContent()` and it does the right thing.

Reply:
I have a massive spreadsheet that maps all the attributes returned from the API for various content types and maps them to the dataset model, which is a superset w/ CPs, and such.